migrator/migrations/m_192_to_m_193_report_config_migrations/migration_impl.go

```diff
@@ -0,0 +1,221 @@
+package m192tom193
+
+import (
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/generated/storage"
+	updatedSchema "github.com/stackrox/rox/migrator/migrations/m_192_to_m_193_report_config_migrations/schema"
+	"github.com/stackrox/rox/migrator/types"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/pkg/postgres/pgutils"
+	"github.com/stackrox/rox/pkg/utils"
+	"github.com/stackrox/rox/pkg/uuid"
+	"gorm.io/gorm/clause"
+)
+
+var (
+	log       = logging.LoggerForModule()
+	batchSize = 3
+)
+
+func create_v2_report_config(reportConfigProto *storage.ReportConfiguration) *storage.ReportConfiguration {
+	// clone v1 report config
+	newConfig := reportConfigProto.Clone()
+	// populate id
+	newConfig.Id = uuid.NewV4().String()
+	// assign collection id in resource scope
+	newConfig.ResourceScope = &storage.ResourceScope{
+		ScopeReference: &storage.ResourceScope_CollectionId{CollectionId: reportConfigProto.GetScopeId()},
+	}
+	//set scopeid to empty string so that v2 api does filter out configs as v1
+	newConfig.ScopeId = ""
+	// add vuln report filter to v2 copy of report config
+	vulnFilter := &storage.VulnerabilityReportFilters{
+		Severities: reportConfigProto.GetVulnReportFilters().GetSeverities(),
+		Fixability: reportConfigProto.GetVulnReportFilters().GetFixability(),
+		ImageTypes: []storage.VulnerabilityReportFilters_ImageType{storage.VulnerabilityReportFilters_DEPLOYED,
+			storage.VulnerabilityReportFilters_WATCHED},
+	}
+	if reportConfigProto.GetVulnReportFilters().GetSinceLastReport() {
+		vulnFilter.CvesSince = &storage.VulnerabilityReportFilters_SinceLastSentScheduledReport{
+			SinceLastSentScheduledReport: true,
+		}
+	} else {
+		vulnFilter.CvesSince = &storage.VulnerabilityReportFilters_AllVuln{
+			AllVuln: true,
+		}
+	}
+	newConfig.Filter = &storage.ReportConfiguration_VulnReportFilters{
+		VulnReportFilters: vulnFilter,
+	}
+	return newConfig
+}
+
+func create_notifier(reportConfigProto *storage.ReportConfiguration) *storage.NotifierConfiguration {
+	notifierConfig := &storage.NotifierConfiguration{
+		Ref: &storage.NotifierConfiguration_Id{
+			Id: reportConfigProto.GetEmailConfig().GetNotifierId()},
+		NotifierConfig: &storage.NotifierConfiguration_EmailConfig{
+			EmailConfig: &storage.EmailNotifierConfiguration{
+				MailingLists:  reportConfigProto.GetEmailConfig().GetMailingLists(),
+				CustomSubject: reportConfigProto.GetEmailConfig().GetCustomSubject(),
```

Comment:
No need to set CustomSubject and CustomBody since they would anyways be empty in v1 configs.